docs/users-guide/interactive-dashboards.md

```diff
@@ -0,0 +1,104 @@
+
+You can also use values to construct URLs to external resources.
+
+![Enter a URL](images/interactive-dashboards/enter-a-url.png)
+
+From the **Click behavior** sidebar, select **Go to a custom destination** and link to **URL**. The **Enter a URL to link to** modal will pop up, allowing you to specify a URL, as well as specify the value of a column or dashboard filter. Click on the dropdown menu **Values you can reference** to see your options.
+
+### Use a chart to filter a dashboard
+
+If your dashboard contains at least one filter, you can set things up so that clicking on a chart in the dashboard will update a filter.
+
+When a user clicks on, say, a bar in a bar chart, you could send the value of the clicked bar to the filter, and update cards across the dashboard. We call this functionality **cross-filtering**. You can use this cross-filtering to make a chart behave as kind of "navigation question" that filters data across other cards.
+
+For example, clicking on the `Widget` bar will update the current dashboard's **category** filter to filter for `Widget`:
+
+![Cross-filtering](images/interactive-dashboards/cross-filter.png)
+
+To set up cross-filtering, choose a dashboard filter that you'd like to update on click, and a question to use to update that filter. You can think of this question as your "navigation question". Instead of wiring this navigation question up to the filter, you'll [wire up every other question on the dashboard to the filter](08-dashboard-filters.md). 
+
+Below, we'll use the **Orders by product category question** as our navigation question, so we'll leave this question disconnected from the filter, and connect all the other questions to the **Category** filter.
+
+![Wiring up filter](images/interactive-dashboards/wiring-up-filter.png)
+
+With your filter wired up, stay in dashboard edit mode, and hover over the question you want to use as your navigation question to filter the dashboard. Click on the **click behavior** icon, then select the **Update a dashboard filter**.
+
+Metabase will list the available filters to update below. Here we select the **Category** filter, and supply the value to that filter from the question's `Product -> Category` column.
+
+![Update a dashboard filter](images/interactive-dashboards/update-a-dashboard-filter.png)
+
+Click **Done** in the sidebar, then **Save** your dashboard. 
+
+Now we can use our navigation question (Orders by product category) to interactively filter the data across your dashboard. When people click on a value in the navigation question, Metabase will send the clicked value to the filter, and update every card on the dashboard by filtering them for the clicked value - every card except for the navigation question: Orders by product category. The reason we don't want the navigation question to update is so that we can select a different category to filter the dashboard.
```

Comment:
```suggestion
Now we can use our navigation question (Orders by product category) to interactively filter the data across your dashboard. When people click on a value in the navigation question, Metabase will send the clicked value to the filter, and update every card on the dashboard by filtering them for the clicked value - every card except for the navigation question: Orders by product category. The reason we don't want the navigation question to be filtered is to make sure all of the bars remain present, so that we can click on a different one to select it instead.
```